test/max-requests.js

```diff
@@ -0,0 +1,118 @@
+const del = require('del');
+const mongo = require('emulate-mongo-2-driver');
+const { expect } = require('chai');
+const rp = require('request-promise');
+const enableDestroy = require('server-destroy');
+const apostropheMultisite = require('./app.js');
+const Promise = require('bluebird');
+
+describe('Apostrophe-multisite', function() {
+  describe('#dashboard', function() {
+    const port = 3000;
+    const admin = 'admin';
+    const url = 'site.test';
+    const title = 'New Site';
+    const shortNamePrefix = 'test-multi-';
+
+    let multisite;
+    let sites;
+    let site;
+    let req;
+    let exited;
+
+    const newSite = {
+      title,
+      slug: 'new-site',
+      adminPassword: admin,
+      devBaseUrl: url,
+      hostnamesArray: [
+        {
+          hostname: url
+        }
+      ]
+    };
+
+    before(async () => {
+      // remove temp files
+      await del(['./test/sites/data']);
+
+      // find and remove test dbs
+      const mongodbUrl =
+        process.env.MONGODB_SERVER && process.env.MONGODB_PORT
+          ? `mongodb://${process.env.MONGODB_SERVER}:${process.env.MONGODB_PORT}`
+          : 'mongodb://localhost:27017';
+      const db = await mongo.MongoClient.connect(mongodbUrl);
+      const adminDb = db.admin();
+      const maxRequestsBeforeShutdown = 10;
+      const additionalRequestsBeforeShutdown = 5;
+      const { databases } = await adminDb.listDatabases();
+      for (const db of databases) {
+        if (db.name.match('[^,]*' + shortNamePrefix + '*')) {
+          const client = await mongo.MongoClient.connect(mongodbUrl + '/' + db.name);
+          await client.dropDatabase();
+          console.log('\x1b[36m%s\x1b[0m', `Test db ${db.name} dropped`);
+        }
+      }
+
+      function exit() {
+        // Mock out process.exit for the test
+        exited = true;
```

Comment:
Is this used? Should it be replaced by an assertion or used in an assertion later?

Reply:
Used, and checked now.